data/mods/Magiclysm/items/spell_scrolls.json

```diff
@@ -586,6 +604,15 @@
     "description": "You become wily like a fox.",
     "use_action": { "type": "learn_spell", "spells": [ "foxs_cunning" ] }
   },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_focused_bolt",
+    "//": "Magus spell",
+    "name": { "str": "Scroll of Focused Bolt", "str_pl": "Scrolls of Focused Bolt" },
+    "description": "You focus your mana into a microscopically thin pulsed beam of pure energy, capable of extreme range and precision;  as well as unparalleled penetration, provided proper skill.  Once the invention of a notorious assassin, this spell has slayed many a king and noble.  Passed on only through direct decendants over the ages, though occasionally shared by them to others.",
```

Comment:
```suggestion
    "description": "You focus your mana into a microscopically thin pulsed beam of pure energy, capable of extreme range and precision; as well as unparalleled penetration, provided proper skill.  Once the invention of a notorious assassin, this spell has slayed many a king and noble.  Passed on only through direct descendants over the ages, though occasionally shared by them to others.",
```

Reply:
; doesnt require double space?

Reply:
I guess so, reading from the error message.